Shokofin/Resolvers/VirtualFileSystemService.cs

```diff
@@ -1085,6 +1133,44 @@ private List<string> FindSubtitlesForPath(string sourcePath) {
         return externalPaths;
     }
 
+    private List<string> FindAudioFilesForPath(string sourcePath) {
```

Comment:
Refactor this to use an `ExternalPathParser` like the subtitle finding logic above, but don't use the same instance as the one used in the subtitle finding logic above, as said instance is tuned for finding subtitles and not audio.

Rename `VirtualFileSystemService.ExternalPathParser` (to e.g. `VirtualFileSystemService.ExternalSubtitlePathParser`, etc.), then add a new parser for finding audio files. Using the parser will ensure we only find valid audio files Jellyfin natively support.